tokens/properties/components/Button.json

```diff
@@ -51,143 +51,295 @@
                 "value": "{dimension.spacing.large.value}"
             }
         },
-        "neutral": {
-            "color": {
-                "value": "{font.color.base.value}"
-            },
-            "border-color": {
-                "value": "{border.color.neutral.default.value}"
-            },
-            "background-color": {
-                "value": "{background.color.neutral.default.value}"
-            },
-            "hover": {
+        "icon-gap": {
```

Comment:
Probably not the best choice of words, but spacing was less self-explanatory? Feedback welcome.